includes/Shopping/Woocommerce_Query.php

```diff
@@ -112,9 +130,14 @@ public function get_search( string $search_term ) {
 	 * @param int $image_id Attachment ID.
 	 * @return array
 	 */
-	protected function get_product_images( int $image_id ): array {
+	protected function get_product_image( int $image_id ): array {
 		$url = wp_get_attachment_url( $image_id );
 		$alt = get_post_meta( $image_id, '_wp_attachment_image_alt', true );
```

Comment:
We can remove this line below url early return. No need of the look of alt if no is nothing.